libraries/AP_HAL_ChibiOS/hwdef/PixFlamingo-F767/README.md

```diff
@@ -0,0 +1,180 @@
+# PixFlamingo-F767 Flight Controller
+
+The PixFlamingo-F767 is a flight controller produced by Dheeran Labs.
+Contact dheeranlabs@gmail.com for sales
+
+## Features
+    Processor
+        STM32F767 32-bit processor
+	Onboard Flash: 2048Mbits
+    Sensors
+        Two IMU : ICM42670, MPU6500/ICM20602 Acc/Gyro
+        One Baro : Internal MS5611/BMP280/DPS310 SPI barometer
+	Internal LIS3MDL Compass
+    Power
+        5v input voltage with voltage monitoring
+    Interfaces
+        10x PWM outputs 
+        1x RC input
+        5x UARTs/serial for GPS and other peripherals
+        2x I2C ports for external compass, airspeed, etc.
+	microSD card slot port
+	Internal RGB LED
+	Safety switch port
+	Buzzer port
+        USB-C port
+
+## Connectors
+
+**POWER ADC**
+
+| Pin  |     Signal      | Volt  |
+| :--: | :-------------: | :---: |
+|  1   |     VCC_IN      |  +5V  |
+|  2   |     VCC_IN      |  +5V  |
+|  3   | BAT_CRRENT_ADC  | +3.3V |
+|  4   | BAT_VOLTAGE_ADC | +3.3V |
+|  5   |       GND       |  GND  |
+|  6   |       GND       |  GND  |
+
+**TELEM1**
+
+| Pin  |  Signal  | Volt  |
+| :--: |  :-----: | :---: |
+|  1   |    VCC   |  +5V  |
+|  2   | UART_TX3 | +3.3V |
+|  3   | UART_RX3 | +3.3V |
+|  4   |    CTS   | +3.3V |
+|  5   |    RTS   | +3.3V |
+|  6   |    GND   |  GND  |
+
+**TELEM2**
+
+| Pin  |  Signal  | Volt  |
+| :--: |  :-----: | :---: |
+|  1   |    VCC   |  +5V  |
+|  2   | UART_TX6 | +3.3V |
+|  3   | UART_RX6 | +3.3V |
+|  4   |    CTS   | +3.3V |
+|  5   |    RTS   | +3.3V |
+|  6   |    GND   |  GND  |
+
+**GPS1**
+
+| Pin  |  Signal  | Volt  |
+| :--: |  :-----: | :---: |
+|  1   |    VCC   |  +5V  |
+|  2   | UART_TX1 | +3.3V |
+|  3   | UART_RX1 | +3.3V |
+|  4   | I2C2_SCL | +3.3V |
+|  5   | I2C2_SDA | +3.3V |
+|  6   |    GND   |  GND  |
+
+**GPS2**
+
+| Pin  |  Signal  | Volt  |
+| :--: |  :-----: | :---: |
+|  1   |    VCC   |  +5V  |
+|  2   | UART_TX7 | +3.3V |
+|  3   | UART_RX7 | +3.3V |
+|  4   |     X    |   X   |
+|  5   |     X    |   X   |
+|  6   |    GND   |  GND  |
+
+
+**SAFETY**
+
+| Pin  |      Signal      | Volt  |
+| :--: | :-----------:    | :---: |
+|  1   |     SAFETY_SW    | +3.3V |
+|  2   |   SAFETY_SW_LED  | +3.3V |
+|  3   |      3V3_OUT	  | +3.3V |
+|  4   |      BUZZER+     | +3.3V |
+|  5   |      BUZZER-     | GND   |
+
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB (OTG1)
+ - SERIAL1 -> UART3 (TELEM1) with CTS/RTS DMA Enabled
+ - SERIAL2 -> UART6 (TELEM2) with CTS/RTS DMA Enabled
```

Comment:
```suggestion
 - SERIAL2 -> UART6 (TELEM2) with CTS/RTS No DMA Enabled
```

Reply:
@IamPete1 can you rebase

Reply:
might as well wait until all the requested changes are addressed